index.html

```diff
@@ -70,21 +76,90 @@ <h4 class="subtitle is-5">Get in touch</h4>
         </div>
         <div>
-            <a href="https://discord.com/invite/5TZjfjs?fbclid=IwAR002nXnqT7St0j6Hf2c-NQgBnWeqfNY70KYaFoDQ1Tlr9OtP6ud_At14o0" target="_blank"><i class="fa fa-comments"></i></a>
+          <a href="https://discord.com/invite/5TZjfjs?fbclid=IwAR002nXnqT7St0j6Hf2c-NQgBnWeqfNY70KYaFoDQ1Tlr9OtP6ud_At14o0"
+            target="_blank"><i class="fa fa-comments"></i></a>
         </div>
         <div>
           <a href="https://www.instagram.com/alfihriyaacademy/" target="_blank"><i class="fa fa-instagram"></i></a>
         </div>
+      </div>
+
     </div>
+  </section>
+
+  <!-- contributors -->
+  <section class="section" id="contributors">
+    <div class="container">
+      <div class="section-contact-heading ">
+        <h3 class="title is-2">Contributors</h3>
+      </div>
+
+      <div class="social-media-links">
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+        <ul class="contributor">
+          <li>
+            <a href="https://github.com/tixboubou"><img src="https://avatars0.githubusercontent.com/u/6819197"
+                alt="Avatar" class="avatar"></a>
+          </li>
+          <Li>
+            <a href="https://github.com/tixboubou">tixboubou</a>
+          </Li>
+        </ul>
+      </div>
 
```

Comment:
should we use the same code, and put our name instead of yours?... I used the indext.hmlt but it didn't work for me?

Reply:
Oui hit 3la makanden rah dar `Li` b majuscule